src/test/java/uk/gov/hmcts/dm/security/domain/DomainPermissionEvaluatorTests.java

```diff
@@ -233,7 +233,7 @@ public void testAsNotCreatorAndNotTestWorkerIdontHavePermissions() {
     public void testRolesReadPermissionsWithClassificationRestrictedAndMatchingRoles() {
         StoredDocument storedFile = new StoredDocument();
         storedFile.setCreatedBy("nobody");
-        storedFile.setRoles(Arrays.asList(new String[] { "allowingrole" }).stream().collect(Collectors.toSet()));
+        storedFile.setRoles(Arrays.stream(new String[] { "allowingrole" }).collect(Collectors.toSet()));
```

Comment:
there should be something like "Set.of("allowingrole")" can be varable and use in all other tests